File: mckp-19/src/main/java/Main.java
import problem.MultipleChoiceKnapsackProblem;
import problem.OneOrNoneForClass;

public class Main {
    public static void main(String[] args) {
        MultipleChoiceKnapsackProblem multipleChoiceKnapsackProblem = new MultipleChoiceKnapsackProblem(new OneOrNoneForClass());
        multipleChoiceKnapsackProblem.solveProblem();
    }
}

File: mckp-19/src/c++/Main.cpp
/**
 * Main class to run the project.
 */

#include "MultipleChoiceKnapsackProblem.h"
#include "OneOrNoneForClass.h"

using namespace std;

using MultipleChoiceKnapsackProblem = problem::MultipleChoiceKnapsackProblem;
using OneOrNoneForClass = problem::OneOrNoneForClass;

int main(int argc, char const* argv[])
{
	MultipleChoiceKnapsackProblem *multipleChoiceKnapsackProblem = new MultipleChoiceKnapsackProblem(new OneOrNoneForClass());
	multipleChoiceKnapsackProblem->solveProblem();

	return 0;
}
